index.html

```diff
@@ -0,0 +1,579 @@
+            normal
+            cosmic history was still ahead, resulting in today’s observable universe, a patch of sky and stars
+            billions
+            of light-years across. “It’s often said that there is no such thing as a free lunch,” Dr. Guth likes to
+            say,
+            “but the universe might be the ultimate free lunch.”
+    </article>
+
+    <article class="article-h">
+        Make that free lunches. Most of the hundred or so models resulting from Dr. Guth’s original vision
+            suggest
+            that inflation, once started, is eternal. Even as our own universe settled down to a comfortable homey
+            expansion, the rest of the cosmos will continue blowing up, spinning off other bubbles endlessly, a
+            concept
+            known as the multiverse.
+            <br> <br>
+            So the future of the cosmos is perhaps bright and fecund, but do not bother asking about going any
+            deeper
+            into the past.
+            <br> <br>
+            We might never know what happened before inflation, at the very beginning, because inflation erases
+            everything that came before it. All the chaos and randomness of the primordial moment are swept away,
+            forever out of our view.
+        “If you trace your cosmic roots,” said Abraham Loeb, a Harvard-Smithsonian astronomer who was not part of
+        the team, “you wind up at inflation.”
+    </article>
+
+    <article class="weekender">
+        <h2>The Weekender</h2>
+        <p>Our editors have handpicked the week’s best stories for you to enjoy in an easy-to-read format.</p>
+        <form action="explore"></form>
+        <input class="explore" type="button" value="Explore The Weekender">
+        <img class="sky" src="images/merlin_.jpg" alt="image of a cliff">
+    </article></div>
+        <div class="right"><aside>
+            <div class="editors-pick">
+                    <ul class="e-picklist">
+                        <li><h3 class="picklist">Editor's Pick</h3></li>
+                <li><a href="#">
+                        <img class="illustrations" src="images/17money-alt-square640.jpg" alt="">
+                        <p>Where Are All The Wild Things,
+                            Daddy?</p>
+                    </a></li>
+                <li><a href="#">
+                        <img class="illustrations" src="images/20MODERN-FATHERSDAY-square640.jpg" alt="">
+                        <p>From the Heart to Higher Education: The 2021 College <br> Essays on
+                            Money</p>
+                    </a></li>
+    
+                <li><a href="#">
+                        <img class="illustrations" src="images/18surfacing-pet-tube-thmpsn1-square640.jpg" alt="">
+                        <p>The Internet’s Exotic Pets, and the People Who Love Them</p>
+                    </a></li>
+                </ul> </div>
+        </aside></div>
+    </div>
```

Comment:
- [ ] Some parts of this section doesn't appear on your live page, please work on that and try make it similar to the [original page](https://bit.ly/3dbcP6X)